src/libraries/Common/src/System/Text/Json/PooledByteBufferWriter.cs

```diff
@@ -76,11 +76,7 @@ public override Span<byte> GetSpan(int sizeHint = MinimumBufferSize)
         public override async ValueTask<FlushResult> FlushAsync(CancellationToken cancellationToken = default)
         {
             Debug.Assert(_stream is not null);
-#if NET
             await _stream.WriteAsync(WrittenMemory, cancellationToken).ConfigureAwait(false);
```

Comment:
I think there are a few other locations where we use `Stream.WriteAsync` in the project. Any chance you could replace them there too?